module/applications/item/ability-use-dialog.mjs

```diff
@@ -48,7 +49,7 @@ export default class AbilityUseDialog extends Dialog {
       consumeResource: resource.target && (!item.hasAttack || (resource.type !== "ammo")),
       consumeUses: uses.per && (uses.max > 0),
       canUse: recharges ? recharge.charged : sufficientUses,
-      createTemplate: game.user.can("TEMPLATE_CREATE") && item.hasAreaTarget,
+      createTemplate: game.user.can("TEMPLATE_CREATE") && (config?.createMeasuredTemplate ?? item.hasAreaTarget),
```

Comment:
Would this not cause an issue if passing `createMeasuredTemplate: true` while `item.hasAreaTarget` is `false`?
It seems like the purpose of the `config` object in this context is to explicitly only disable some properties in the `data` object, but here it is taking priority. Maybe adding `config.createMeasuredTemplate !== false` instead would work (or using `mergeObject`), like,
```js
createTemplate: game.user.can("TEMPLATE_CREATE") && item.hasAreaTarget && (config.createMeasuredTemplate !== false),
```

Reply:
A change was pushed to make sure that it cannot be true if is not an AOE